pkg/virt-operator/resource/generate/rbac/cluster.go

```diff
@@ -138,74 +175,73 @@ func newAdminClusterRole() *rbacv1.ClusterRole {
 		Rules: []rbacv1.PolicyRule{
 			{
 				APIGroups: []string{
-					GroupNameSubresources,
+					virtv1.SubresourceGroupName,
 				},
 				Resources: []string{
-					"virtualmachineinstances/console",
-					"virtualmachineinstances/vnc",
-					"virtualmachineinstances/vnc/screenshot",
-					"virtualmachineinstances/portforward",
-					VMInstancesGuestOSInfo,
-					VMInstancesFileSysList,
-					VMInstancesUserList,
+					ApiVMInstancesConsole,
+					ApiVMInstancesVNC,
+					ApiVMInstancesVNCScreenshot,
+					ApiVMInstancesPortForward,
+					ApiVMInstancesGuestOSInfo,
+					ApiVMInstancesFileSysList,
+					ApiVMInstancesUserList,
 				},
 				Verbs: []string{
 					"get",
 				},
 			},
 			{
 				APIGroups: []string{
-					GroupNameSubresources,
+					virtv1.SubresourceGroupName,
 				},
 				Resources: []string{
-					"virtualmachineinstances/pause",
-					"virtualmachineinstances/unpause",
-					"virtualmachineinstances/addvolume",
-					"virtualmachineinstances/removevolume",
-					"virtualmachineinstances/freeze",
-					"virtualmachineinstances/unfreeze",
-					"virtualmachineinstances/softreboot",
+					ApiVMInstancesPause,
+					ApiVMInstancesUnpause,
+					ApiVMInstancesAddVolume,
+					ApiVMInstancesRemoveVolume,
+					ApiVMInstancesFreeze,
+					ApiVMInstancesUnfreeze,
+					ApiVMInstancesSoftReboot,
 				},
 				Verbs: []string{
 					"update",
 				},
 			},
 			{
 				APIGroups: []string{
-					GroupNameSubresources,
+					virtv1.SubresourceGroupName,
 				},
 				Resources: []string{
-					"virtualmachines/expand-spec",
-					"virtualmachines/portforward",
+					ApiVMExpandSpec,
+					ApiVMPortForward,
 				},
 				Verbs: []string{
 					"get",
 				},
 			},
 			{
 				APIGroups: []string{
-					GroupNameSubresources,
+					virtv1.SubresourceGroupName,
 				},
 				Resources: []string{
-					"virtualmachines/start",
-					"virtualmachines/stop",
-					"virtualmachines/restart",
-					"virtualmachines/addvolume",
-					"virtualmachines/removevolume",
-					"virtualmachines/migrate",
-					"virtualmachines/memorydump",
-					"virtualmachines/addinterface",
```

Comment:
This seems to be a mistake, `addinterface` is missing from the refactoring, please put that back in.

Reply:
See commit message for details. It was removed on purpose because the backport in 73c3a5ec053635e47e41f7cf738fe0f967e2f05d missed to remove it.

Reply:
Let me recheck, but I think it was missed in a previous backport...

Reply:
My mistake, mixed up the commits removing it from VM/VMI. Readded it to the backport.